UICatalog/Scenarios/SingleBackgroundWorker.cs

```diff
@@ -9,7 +9,7 @@ namespace UICatalog.Scenarios;
 
 [ScenarioMetadata ("Single BackgroundWorker", "A single BackgroundWorker threading opening another Toplevel")]
 [ScenarioCategory ("Threading")]
-[ScenarioCategory ("Overlapped")]
+[ScenarioCategory ("Arrangement")]
 [ScenarioCategory ("Runnable")]
 public class SingleBackgroundWorker : Scenario
```

Comment:
The `MenuBar` of the Worker window isn't showing and can't be closed.

Reply:
Can you  please help me debug?

Reply:
I submitted the PR https://github.com/tig/Terminal.Gui/pull/42 to fix this.